### PR TITLE
osslsigncode: new package

### PR DESCRIPTION
**Pull request by elieux**

### CODE REVIEW

File: mingw-w64-osslsigncode/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Maintainer: Alexey Pavlov <alexpux@gmail.com>
+
```
Comment (mingwandroid) [inline mingw-w64-osslsigncode/PKGBUILD]: You are the maintainer surely David?

File: mingw-w64-osslsigncode/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+_realname=osslsigncode
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
```
Comment (mingwandroid) [inline mingw-w64-osslsigncode/PKGBUILD]: pkgbase missing

**Comment by elieux** [comment]
Heh, I guess I was too quick to copypaste. New commit coming soon.

**Comment by mingwandroid** [comment]
Also, feel free to commit yourself. If you make mistakes occasionally (as we all do) then they can be caught later, it's no big deal.

**Comment by elieux** [comment]
Updated.

I was thinking of commiting myself, but the thing I'm not sure about is how @alexpux determines which packages need building. It wouldn't be very useful if I commited and then nobody built the package. Along those lines, I'd like to open a discussion (again) about building packages by other contributors.

**Pull request merged**